### PR TITLE
Option for using secondary task

### PR DESCRIPTION
**Pull request by MarkusJohansen**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
In scenes where you have two task that you would like to start individually, you can now control spawners and conveyours by using the `isSecondaryTaskOn` boolean in `GameManager`

* **What is the current behavior?** (You can also link to an open issue here)
You must control all QA conveyours with Sorting machine red-button
closes: #428 

* **What is the new behavior?** (if this is a feature change)
The two tasks in QA can be controlled separately

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Changes in QA station.

### CODE REVIEW

File: Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab
Comment (jonp-h) [inline Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab:1]: Does this need to be it's own prefab? Or is a quick "setting change" when placing the original prefab enough?

**Review by jonp-h — APPROVED** [review]
Read comments, else good to merge

File: Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab
Comment (MarkusJohansen) [reply Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab:1]: Hmm, i only changed the script really, it does not need to be a separate prefab

Comment (MarkusJohansen) [reply Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab:1]: we should change the original prefab

Comment (MarkusJohansen) [reply Assets/FishFactory/Resources/Prefabs/Conveyor_type2.prefab:1]: Ahh now i see, i didnt create a new prefab, but i think that these are the "new" prefabs that @mefriis exchanged our old ones with. If we shall remove the old ones, we can do it in the final file cleanup, before project deadline.

**Pull request merged**